parm/aero/obs/config/viirs_n20_aod.yaml

```diff
@@ -14,6 +14,7 @@ obs space:
   channels: 4
 get values:
   interpolation method: barycentric
```

Comment:
Ha! I wasn't aware this was working @CoryMartin-NOAA. Nice!

Reply:
Well it wasn't, but there is an OOPS bugfix PR so I figured might as well.